spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -329,9 +329,22 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     castTest(generateFloats(), DataTypes.createDecimalType(10, 2))
   }
 
-  ignore("cast FloatType to StringType") {
+  test("cast FloatType to StringType") {
     // https://github.com/apache/datafusion-comet/issues/312
```

Comment:
We should remove the references to the issue now that we are resolving the issue

```suggestion
```